src/targets/gpu/hiprtc/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ add_executable(migraphx-hiprtc-driver
     main.cpp
 )
 rocm_clang_tidy_check(migraphx-hiprtc-driver)
+if(WIN32)
+    target_link_options(migraphx-hiprtc-driver PRIVATE -Xlinker /stack:4194304)
```

Comment:
I think this option is only valid when using MSVC linker. We should probably check for that.

Reply:
There is no need. The llvm-ld.exe is just a frontend, and on Windows, it invokes the MSVC linker (link.exe).

Reply:
The only exception is when MINGW (or CYGWIN) is used then the GNU linker is used.

Reply:
Sure, but we could add a check for GCC and skip adding this flag so it wont break any users that might want to build with gcc in the future.

Reply:
@pfultz2 done